src/components/button/Button.tsx

```diff
@@ -82,6 +82,7 @@ class Button extends React.Component<ButtonProps> {
                 'is-disabled': isDisabled,
                 'is-loading': isLoading,
                 'is-selected': isSelected,
+                'bdl-is-disabled': isDisabled,
```

Comment:
@greg-in-a-box

Regarding your question,
I checked the commit log, and found this.
https://github.com/swfree/box-ui-elements/pull/2

A "bdl-" namespace was added to solve the conflicting classnames.
There's a [script](https://github.com/box/box-ui-elements/blob/master/scripts/utils/bdlClassnameManager) for this, but it doesn't cover all cases.

I was struggling to replace `is-disabled` with `bdl-is-disabled`, but it would be a breaking change here. Or should I just remove [bdl-is-disabled](https://github.com/box/box-ui-elements/blob/master/src/styles/common/_buttons.scss#L175)? What do you think?

Reply:
I don't think you would want to straight replace `is-disabled` unless you were going to immediately update the parents apps that reference this class.

E.g. EUA, Admin Console, etc. have SCSS files that use this class, so that would break their UI when you bump the BUIE version in those apps.